package.json

```diff
@@ -14,19 +15,10 @@
     "ios",
     "android"
   ],
-  "author": {
-    "name": "Ladislav Martincik",
-    "email": "ladislav.martincik@gmail.com",
-    "url": "https://github.com/martincik"
-  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/slowpath/react-native-hockeyapp"
-  },
-  "bugs": {
-    "url": "https://github.com/slowpath/react-native-hockeyapp/issues"
+    "url": "https://github.com/peartherapeutics/react-native-hockeyapp"
```

Comment:
if we're publishing this in public, we should. @jasonfma do you mind if I turn it on for this repo